app/controllers/api/v1/application_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::ApplicationsController < ApplicationController
+
+  def show
+    stop = Stop.find(params[:stop_id])
+    review = stop.reviews.find(params[:id])
+    render review
+    render stop
+  end
```

Comment:
hmmm, Is this supposed to be `applications` controller, or Api::V1::StopsController? 

Also you should only have one render per method. You will likely want to use a serializer here to pass over the stop and the reviews.
Also make sure to change `review` to `reviews`